<?xml version="1.0"?>
<launch>
  <include file="$(find ur_modern_driver)/launch/ur10_bringup.launch">
    <arg name="robot_ip" value="192.38.66.226" />    	    
  </include>
   <node name="chess_pickup_sequence" pkg="rob_auto_cal" type="chess_pickup_sequence" output="screen" args="/misc/shome/ex09/Thesis/Z_file" />
</launch>